drivers/gles2/rasterizer_storage_gles2.cpp

```diff
@@ -6294,6 +6301,17 @@ void RasterizerStorageGLES2::initialize() {
 	config.render_to_mipmap_supported = config.extensions.has("GL_OES_fbo_render_mipmap") && config.extensions.has("GL_EXT_texture_lod");
 #endif
 
+	// If the desktop build is using S3TC, and you export / run from the IDE for android, if the device supports
+	// S3TC it will crash trying to load these textures, as they are not exported in the APK. This is a simple way
+	// to prevent Android devices trying to load S3TC, by faking lack of hardware support.
+
+	// Switch: this happens on Horizon too.
+#ifndef TOOLS_ENABLED
+#if defined ANDROID_ENABLED || defined HORIZON_ENABLED
+	config.s3tc_supported = false;
```

Comment:
This is happening in two different places.

Reply:
- [ ] Remove this block